identity-service/src/routes/notifications.js

```diff
@@ -382,10 +382,38 @@ module.exports = function (app) {
         announcementsAfterFilter
       )
 
+      let playlistUpdates = []
+      const user = await models.User.findOne({
+        attributes: ['walletAddress'],
+        where: { id: userId },
+        raw: true
```

Comment:
same comment about raw here too. what it do

Reply:
yup. will probably be removed too. it's this thing about models returning the raw json vs an object with `dataValues`. going back to check :)